build-configs/ThemeType.ts

```diff
@@ -9,6 +9,20 @@ export type PaletteColor = SimplePaletteColor & {
   contrastText: string
 }
 
+export type ActionColor = {
```

Comment:
❓ where do you have also these action states? Is the source from figma?
What is `activatedOpacity`?